src/virtual-select.js

```diff
@@ -448,6 +448,7 @@ export class VirtualSelect {
     this.$toggleAllCheckbox = this.$dropboxContainer.querySelector('.vscomp-toggle-all-checkbox');
 
     this.addEvent(this.$searchInput, 'input', 'onSearch');
+    this.addEvent(this.$searchInput, 'change', 'toPreventEvent');
```

Comment:
Add unit tests to verify that the change event is no longer fired twice when selecting items via the search input.
They should include all scenarios, like when you are in popup mode, inline mode, always opened, setting values programmatically, etc. I would request this since adding this sort of workaround that at first glance seem to be siple can be often dangerous by adding unwanted side effects, so regression testing should be guaranteed

Reply:
I added a basic test over the select from the examples to cover this error. Of course, it would be desirable to add other states, but I'm afraid this is too complicated for me. It would be good if someone added this